app/src/main/java/skills/future/planer/ui/month/CustomMultipleDotSpan.java

```diff
@@ -0,0 +1,81 @@
+package skills.future.planer.ui.month;
+
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.text.style.LineBackgroundSpan;
+
+/**
+ * Class implements LineBackgroundSpan and draws dots under dates
+ *
+ * @author Paweł Helisz
+ * @version 0.2.2
+ * @since 0.2.1
+ */
+public class CustomMultipleDotSpan implements LineBackgroundSpan {
+
+    /**
+     * dot radius
+     */
+    private final float radius;
+    /**
+     * number of dots under date
+     */
+    private final int dots_number;
+    /**
+     * colors of dots
+     */
+    int[] color = {
+            Color.rgb(0, 0, 255),
+            Color.rgb(0, 255, 0),
+            Color.rgb(255, 0, 0),
+            Color.rgb(255, 0, 255),
+    };
+
+    /**
+     * Constructor
+     * @param radius dot radius
+     * @param dots_number number of dots under date
+     */
+    public CustomMultipleDotSpan(float radius, int dots_number) {
+        this.radius = radius;
```

Comment:
Jeżeli wszystkie kropki mają ten sam rozmiar to dodaj ten rozmiar jako pole final, a nie jako nie potrzeby parametr

Reply:
To też nie wiadomo, zostawiłem bo może będzie potrzeba zmieniania parametrów dla ikonek